src/chapters/sql-part1/exercises.rst

```diff
@@ -0,0 +1,215 @@
+   reference `the documentation <https://dev.mysql.com/doc/workbench/en/wb-home.html>`__.
+
+Table Setup
+-----------
+
+Imagine that you want to apply your new SQL skills to bring some order to your
+garden. For these exercises, you will use a ``seeds`` table to store
+information for your garden hopes. The table will have 4 columns:
+
+#. ``seed_id``: This is a unique number assigned to each row in the ``seeds``
+   table.
+#. ``crop``: Identifies what the seeds grow (e.g. yellow bell peppers).
+#. ``encourages``: Identifies the benefits of planting the crop (e.g. nitrogen
+   fixation, attracts bees, etc.)
+#. ``use_by``: The year the seeds expire.
+
+The ``crop`` and ``encourages`` columns will hold ``VARCHAR`` data types, and
+each must include a parameter that specifies the maximum number of characters.
+``seed_id`` and ``use_by`` will be integers.
+
+Paste the following script into the query tab, then run it.
+
+.. sourcecode:: SQL
+   :linenos:
+
+   CREATE TABLE seeds (
+      seed_id INTEGER PRIMARY KEY AUTO_INCREMENT,
+      crop VARCHAR(40),
+      encourages VARCHAR(80),
+      use_by INTEGER
+   );
+
+Line 2 establishes ``seed_id`` as the unique identifier---the
+``PRIMARY KEY``---for each record in the table. ``AUTO_INCREMENT`` assigns
+every new entry a different integer value.
+
+Properly done, you should now see the ``seeds`` table within your database.
+
+.. figure:: ./figures/seedsTableFileTree.png
+   :alt: File tree for the seeds table.
+
+Create
+------
+
+Open up a new query tab for the SQL commands you code in this section.
+
+.. admonition:: Tip
+
+   MySQLWorkbench allows you to run one SQL command, a set of commands, or all
+   of the commands listed in the editor pane. Hover over each lightning bolt
+   icon in the panel to see these options.
+
+   .. figure:: ./figures/workbenchBoltIcons.png
+      :alt: Lightning bold icons in MySQLWorkbench.
+
+#. Use the following SQL command to add a new entry to the ``seeds`` table.
+
+   .. sourcecode:: SQL
+      :linenos:
+
+      INSERT INTO seeds (crop, encourages, use_by)
+      VALUES ("Agastache", "bees & hummingbirds", 2020);
+
+   Notice that you do NOT need to provide a value for ``seed_id``, since it is
+   set up to auto-increment every time a new record is created.
+
+   .. admonition:: Tip
+
+      At any time, you can confirm that a table contains data by clicking on
+      the table icon next to its name.
+
+      .. figure:: ./figures/seedsTableCheck.png
+         :alt: View table contents button.
+
+#. Add another new entry to the ``seeds`` table, choosing your own values for
+   the columns.
+#. To add values to only *some* of the columns of the table, simply omit those
+   column names and values from the SQL command.
+
+   .. sourcecode:: SQL
+      :linenos:
+
+      INSERT INTO seeds (crop, use_by)
+      VALUES ("Sun Gold Tomato", 2022);
+
+#. Add 3 - 5 more records to the ``seeds`` table.
```

Comment:
Do they have to add values to all of the columns?

Reply:
Good point.  I added words to clarify this.